src/lavatory/credentials.py

```diff
@@ -15,3 +15,20 @@ def load_credentials():
             raise MissingEnvironmentVariable(key.upper())
 
     return credentials
+
+def load_slack_credentials():
+    """Get slack credentials from environment variables
+    
+    Returns False if no credentials set.
+
+    """
+
+    slack_credentials = {
+        "api_token": os.getenv('SLACK_API_TOKEN', False),
```

Comment:
There are some documentation pieces that would need to be update to reflect these new environment variables.

Reply:
fixed! :)